packages/interface/src/features/ballot/components/BallotConfirmation.tsx

```diff
@@ -100,12 +111,60 @@ export const BallotConfirmation = ({ pollId }: IBallotConfirmationProps): JSX.El
 
         <Notice
           content={
-            round?.votingEndsAt ? format(round.votingEndsAt, "d MMM yyyy hh:mm") : "The date would be announced soon."
+            round?.votingEndsAt
+              ? `Results will be available after tallying - ${format(round.votingEndsAt, "d MMM yyyy hh:mm")}`
+              : "The date would be announced soon."
           }
-          title="Results will be available after tallying."
-          variant="block"
         />
 
+        <Card variant="invert">
+          <div className="text-center">
+            <b className="text-2xl uppercase">Share that you have voted in {round?.roundId} round.</b>
+          </div>
+
+          <div className="mt-2 flex w-full flex-col items-center justify-center gap-2 sm:flex-row sm:gap-4">
+            <Button
+              as={Link}
+              href={`https://x.com/intent/tweet?text=I+successfully+submit+my+vote+in+${round?.roundId}+round.`}
```

Comment:
Maybe the text could be a constant so that it can be more easily updated (then we gotta think of a way to get the round name at least for the future but that's for another pr imo)

Reply:
same reply as above

Reply:
you have 3 times the same text, so if you make a variable then you can just change it in one place rather than 3